shared/common-adapters/tab-bar.native.js

```diff
@@ -85,10 +78,10 @@ class TabBar extends Component {
 
   _labels (): Array<React$Element> {
     // TODO: Not sure why I have to wrap the child in a box, but otherwise touches won't work
-    return (this.props.children || []).map((item, i) => (
-      <TouchableWithoutFeedback key={item.props.label || i} onPress={item.props.onClick || (() => {})}>
-        <Box style={item.props.containerStyle}>
-          {item.props.tabBarButton || <SimpleTabBarButton tabWidth={this.props.tabWidth} label={item.props.label} selected={item.props.selected} underlined={this.props.underlined} />}
+    return (this.props.children || []).map((item: {props: ItemProps}, i) => (
+      <TouchableWithoutFeedback key={`${i}-${item.props.label}`} onPress={item.props.onClick || (() => {})}>
```

Comment:
Or maybe a key can be optionally provided. That's the approach that I'm going to take in a new commit.